packages/gatsby-source-nacelle/src/source-nodes/index.js

```diff
@@ -82,10 +83,11 @@ module.exports = async function ({
                 let images = findNestedImages(node.remoteFields, [
                   'remoteFields'
                 ])?.filter((val) => val);
-                // create remote image file nodes for each field
-                await Promise.all(
-                  fetchRemoteImageNodesForContent(node, images, gatsbyApi)
-                );
+                await findNestedReferences(node.remoteFields, node),
+                  // create remote image file nodes for each field
+                  await Promise.all(
+                    fetchRemoteImageNodesForContent(node, images, gatsbyApi)
+                  );
```

Comment:
Looks like there was a suspect `,`:

```suggestion
                await findNestedReferences(node.remoteFields, node)
                // create remote image file nodes for each field
                await Promise.all(
                  fetchRemoteImageNodesForContent(node, images, gatsbyApi)
                );
```

Reply:
This should be fixed now! Thanks for the catch!